azurelinuxagent/ga/cgroupcontroller.py

```diff
@@ -164,7 +164,7 @@ def _found_cgroup_procs(file):
                 return True
 
             # In v2, the cgroup.procs file is present in the scope cgroup for extensions
-            for cgroup_file in glob.iglob(os.path.join(self.path, "*/cgroup.procs")):
+            for cgroup_file in glob.iglob(os.path.join(self.path, "*.scope/cgroup.procs")):
```

Comment:
is it possible to match more specifically the scopes we create?

Reply:
"*" would match any of those 3 examples, and any other scope, if there is ever one.

my ask is if we can make it match only the scope that we know we created

Reply:
actually, never mind, sorry, your original change was ok. we want to continue tracking as long as there are any processes, no matter who created them, so * is OK